bin/dynflow-executor

```diff
@@ -2,45 +2,74 @@
 
 require 'optparse'
 
-options = { foreman_root: Dir.pwd }
+class ArgvParser
+  attr_reader :options, :command
 
-opts = OptionParser.new do |opts|
-  opts.banner = <<BANNER
+  def initialize(argv, file)
+    @options = { foreman_root: Dir.pwd }
+
+    opts = OptionParser.new do |opts|
+      opts.banner = banner(file)
+
+      opts.on('-h', '--help', 'Show this message') do
+        puts opts
+        exit 1
+      end
+      opts.on('-f', '--foreman-root=PATH', "Path to Foreman Rails root path. By default '#{@options[:foreman_root]}'") do |path|
+        @options[:foreman_root] = path
+      end
+      opts.on('-c', '--executors-count=COUNT', 'Number of parallel executors to spawn. Overrides EXECUTORS_COUNT environment varaible.') do |count|
+        @options[:executors_count] = count.to_i
+      end
+      opts.on('-m', '--memory-limit=SIZE', 'Limits the amount of memory an executor can consume. Overrides EXECUTOR_MEMORY_LIMIT environment varaible. You can use kb, mb, gb') do |size|
+        @options[:memory_limit] = begin
+          size.to_gb.gigabytes
```

Comment:
This logic needs to move to `daemon.rb` file, as otherwise passing 2GB from the sysconfig file doesn't work